### PR TITLE
Improve WorkHub client

### PR DESCRIPTION
**Pull request by eugenijusr**
This PR updates the WorkHub client with the latest bug fixes and features.

### CODE REVIEW

**Comment by eugenijusr** [comment]
@deviantbits Can you take a quick peek?

**Comment by deviantbits** [comment]
Sure.

File: lib/workhub/validation_error.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+module WorkHub
+  class ValidationError < StandardError
+    attr_accessor :errors
```
Comment (deviantbits) [inline lib/workhub/validation_error.rb]: I don't particularly like the semantics here. It's a `ValidationError` but it also has `:errors` and gets initialized `with_errors`. How about calling the individual `:errors` "field errors" or "attribute errors"?

**Comment by deviantbits** [comment]
@eugenijusr Looks good, but I have one comment. Otherwise, it's `ready to merge`.

File: lib/workhub/validation_error.rb (hunk 1 above)
Comment (eugenijusr) [inline lib/workhub/validation_error.rb]: Maybe just `items` instead?

Comment (deviantbits) [inline lib/workhub/validation_error.rb]: `items` is just fine. Good you fixed the instantiation, btw.

**Pull request merged**